resources/views/components/menu-dropdown.blade.php

```diff
@@ -1,15 +1,21 @@
+@if($nested)
+    @include(backpack_view('components.partials.nested-menu-dropdown'))
+@else
 <li class="nav-item dropdown">
```

Comment:
For some reason, if we have a nested dropdown, it always shows up as active in Tabler:

![CleanShot 2023-08-31 at 09 01 00](https://github.com/Laravel-Backpack/theme-tabler/assets/1032474/9342d553-c61d-4e96-9b0a-d5bf77d582d3)

In our demo, I've changed the News to a dropdown (instead of a header):
```html
    <x-backpack::menu-dropdown title="News" icon="la la-newspaper-o" nested="true">
        <x-backpack::menu-dropdown-item title="Articles" icon="la la-newspaper-o" :link="backpack_url('article')" />
        <x-backpack::menu-dropdown-item title="Categories" icon="la la-list" :link="backpack_url('category')" />
        <x-backpack::menu-dropdown-item title="Tags" icon="la la-tag" :link="backpack_url('tag')" />
    </x-backpack::menu-dropdown>
```

it looks and works PERFECT in CoreUIv2 and CoreUIv4.

And it works perfect in Tabler too. But it's always "active". Note that I'm on the FileManager page, which has nothing to do with News, yet News shows up as active and there's the `active` class on that dropdown-toggle even if it shouldn't be there:

![CleanShot 2023-08-31 at 09 03 50](https://github.com/Laravel-Backpack/theme-tabler/assets/1032474/1cab533d-c8d0-44b1-8d43-49725804beae)

I suspect this is because of the menu-activation-js, which is different in every theme?